src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx

```diff
@@ -264,6 +267,28 @@ export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
                             onShowRelatedClick={onShowRelated}
                         />
                     )}
+
+                    {showDataEntities && availableScopes?.includes(EntryScope.Presentation) && (
```

Comment:
Is the "Limited Viewer" role not enough to use reports? Reports are similar to dashboards and should have a similar access model, isn't it?

Reply:
Fixed